tests/serverpod_test_server/test_integration/authentication/auth_config_changes_test.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:serverpod_auth_server/module.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var userName = 'test';
+  var email = 'test@serverpod.dev';
+  var password = 'password';
+  var session = await IntegrationTestServer().session();
+
+  var codeLengthsToTest = [-16, -8, -4, -1, 0, 1, 4, 16];
+
+  test(
+      'Not setting validationCodeLength will generate a code with default length 8',
```

Comment:
Suggestion: Following the `Given, When, Then` pattern a suggestion for this description could be "Given no validation code length configuration when creating an account then validation code has the default length of 8.".